src/main/java/org/fcrepo/auth/webac/WebACAuthorizationDelegate.java

```diff
@@ -41,6 +56,24 @@
      */
     private static final Logger LOGGER = LoggerFactory.getLogger(WebACAuthorizationDelegate.class);
 
+    private static final Map<String, String> actionMap;
+
+    static {
+        final Map<String, String> map = new HashMap<>();
+        // WEBAC_MODE_READ Permissions
+        map.put(READ, WEBAC_MODE_READ_VALUE);
+        // WEBAC_MODE_WRITE Permissions
```

Comment:
I am referring to the non-standard F4-WebAC modes of `DELETE` and `UPDATE`, not ModeShape actions.
See: https://wiki.duraspace.org/display/FF/Design+-+WebAccessControl+Authorization+Delegate#Design-WebAccessControlAuthorizationDelegate-ProposedRequirements(Phase1)